wiki/Game_modifier/No_Video/id.md

```diff
@@ -0,0 +1,36 @@
+---
+stub: true
+tags:
+  - no video
+  - NV
+  - mod
+  - game modifier
+---
+
+<!-- TODO:
+- add score multiplier, caption, and shortucut key values -->
+
+# No Video (mod)
+
+![Ikon mod NV](/wiki/shared/mods/NV.png "Ikon mod No Video (NV)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+
+## Tentang
+
+- Singkatan: NV
+- Tipe: Spesial
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

Comment:
```suggestion
- Mode-mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

Reply:
didn't we use ```game``` across the board?

Reply:
we already discussed this for specific sub forum to using Mode Permainan instead